src/routes/root.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Navigate } from 'react-router';
+import About from '../components/About';
+import Agencies from '../components/Agencies';
+import Astronauts from '../components/Astronauts';
+import Events from '../components/Events';
+import Home from '../components/Home';
+
+const rootRoutes = [
+  {
+    path: '/',
+    element: <Home />,
+  },
+  {
+    path: '/events',
+    element: <Events />,
+  },
+  {
+    path: '/agencies',
```

Comment:
1. Rename`routes` to `Routes` and `root.tsx` to `Routes.tsx`.
2. Move `Routes` folder to the root of the `components` folder.
3. Create `routes.ts` in the `Routes` folder and move router paths here.

Reply:
Done.
3. routesUrl.ts  was created  instead of routes.ts (because was error in import)